test/interop/python/multilocale/checkMultipleLocales.good

```diff
@@ -0,0 +1,8 @@
+Hello, world! (from task 0 of 2 on locale 0 of 4)
```

Comment:
The order of the output for this test is always guaranteed to be this, even though it's running in a `coforall`?

Reply:
Yup, I added a prediff that sorts the output